artifacts/source_type.py

```diff
@@ -426,14 +426,23 @@ def DeregisterSourceType(cls, source_type_class):
 
   @classmethod
   def GetSourceTypes(cls):
-    """Retriees the source types.
+    """Retrieves the source types.
 
     Returns:
       A list of source types (subclasses of SourceType).
     """
     return cls._source_type_classes.values()
 
   @classmethod
+  def GetSourceTypeIndicators(cls):
```

Comment:
Well the place I was using it internally got refactored out in favor of letting AddSources just raise, but I am still using it externally because GetSourceType returning a list of objects instead of a list of names wasn't as useful.  I added this rather than changing GetSourceType in case that was getting used elsewhere.